R/init.R

```diff
@@ -385,6 +396,54 @@ if (interactive() &&
         page_viewer = page_viewer
       )
 
+      #vsc-r-api
+      RESPONSE_TIMEOUT <- 30
```

Comment:
Does it look more consistent if we keep using the snake_case here?

Also, waiting for 30 seconds until timeout seems a bit too long? Could we use a more reasonable number which is small but sufficient to indicate there might be a problem, e.g. 5 seconds?

Reply:
I am used to using caps for numeric constants but if that's not a style you use no worries. Yeah 30 seconds is definitely too long, 5 sounds good.